docs/sdk/android-async.md

```diff
@@ -45,9 +56,19 @@ AppCenter.isEnabled().thenAccept(new AppCenterConsumer<Boolean>() {
     }
 });
 ```
+```kotlin
+AppCenter.isEnabled().thenAccept(object : AppCenterConsumer<Boolean> {
+    override fun accept(enabled: Boolean?) {
+        Log.d("MyApp", "AppCenter.isEnabled=$enabled")
+    }
+})
```

Comment:
Android Studio suggests converting this to a lambda. I think it might be a good idea to use the suggested code, e.g.
```kotlin
AppCenter.isEnabled().thenAccept { enabled -> Log.d("MyApp", "AppCenter.isEnabled=$enabled") }
```